pulp_smash/utils.py

```diff
@@ -80,3 +82,38 @@ def poll_task(server_config, href):
         # This approach is dumb, in that we don't account for time spent
         # waiting for the Pulp server to respond to us.
         sleep(5)
+
+
+def get_broker(server_config):
+    """Build an object for managing the target system's AMQP broker.
+
+    Talk to the host named by ``server_config`` and use simple heuristics to
+    determine which AMQP broker is installed. If Qpid or RabbitMQ appear to be
+    installed, return a :class:`pulp_smash.cli.Service` object for managing
+    those services respectively. Otherwise, raise an exception.
+
+    Another option for solving this problem is to create a subclass of
+    ``Service``, like so:
+
+    >>> class Broker(Service):
```

Comment:
That sounds good. We used to put links in our code, then one day we switched issue trackers. I'm not sure to recommend to guard against that. What you propose sounds good.